<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CNewUser</class>
 <widget class="QDialog" name="CNewUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adicionar usuario</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lblName">
       <property name="text">
        <string>&amp;Nombre</string>
       </property>
       <property name="buddy">
        <cstring>leName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leName"/>
     </item>
     <item>
      <widget class="QLabel" name="lblNacId">
       <property name="text">
        <string>ID &amp;Nacional</string>
       </property>
       <property name="buddy">
        <cstring>leId</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leId"/>
     </item>
     <item>
      <widget class="QLabel" name="lblBirth">
       <property name="text">
        <string>Fecha de nacimiento</string>
       </property>
       <property name="buddy">
        <cstring>deBirth</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="deBirth">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblCountry">
       <property name="text">
        <string>&amp;País</string>
       </property>
       <property name="buddy">
        <cstring>cbxCountry</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbxCountry">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cuba</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>USA</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Uruguay</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canada</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Armenia</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rusia</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>España</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Polonia</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Francia</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bélgia</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Italia</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Malta</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Marruecos</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UAE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mexico</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Colombia</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblSex">
       <property name="text">
        <string>Se&amp;xo</string>
       </property>
       <property name="buddy">
        <cstring>cbxSex</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbxSex">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Masculino</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Femenino</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Otro</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblPhone">
       <property name="text">
        <string>Telé&amp;fono</string>
       </property>
       <property name="buddy">
        <cstring>lePhone</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lePhone"/>
     </item>
     <item>
      <widget class="QLabel" name="lblRegTime">
       <property name="text">
        <string>&amp;Fecha y hora de registro</string>
       </property>
       <property name="buddy">
        <cstring>dteRegister</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dteRegister">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="2">
      <widget class="QPushButton" name="pbAccept">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLabel" name="lblFacePicture">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="lblShowFace">
       <property name="text">
        <string>Imagen de rostro</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0" rowspan="2">
      <widget class="QLabel" name="lblFullPicture">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblShowFull">
       <property name="text">
        <string>Imagen completa</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblIdPicture">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QToolButton" name="tbtPicture">
       <property name="text">
        <string>Car&amp;gar imagen</string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::ToolButtonPopupMode::MenuButtonPopup</enum>
       </property>
       <property name="arrowType">
        <enum>Qt::ArrowType::NoArrow</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblShowId">
       <property name="text">
        <string>Imagen de ID</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leName</tabstop>
  <tabstop>leId</tabstop>
  <tabstop>deBirth</tabstop>
  <tabstop>cbxCountry</tabstop>
  <tabstop>cbxSex</tabstop>
  <tabstop>lePhone</tabstop>
  <tabstop>dteRegister</tabstop>
  <tabstop>tbtPicture</tabstop>
  <tabstop>pbCancel</tabstop>
  <tabstop>pbAccept</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
